config/semantic-forms/20240110093200-forms.ttl

```diff
@@ -314,7 +314,12 @@ fields:3dd6ed93-40f7-4d70-a6cb-f4de53dc8bfb a form:Field ;
       [ a form:ValidDateTime ;
         form:grouping form:MatchEvery ;
         sh:path ( [ sh:inversePath prov:generated ] dct:subject [ sh:inversePath besluit:behandelt ]  prov:startedAtTime ) ;
-        sh:resultMessage "Geef een geldige datum en tijd op."] ;
+        sh:resultMessage "Geef een geldige datum en tijd op."],
+      [ a form:DateInPast ;
+        form:grouping form:MatchEvery ;
+        sh:path ( [ sh:inversePath prov:generated ] dct:subject [ sh:inversePath besluit:behandelt ]  prov:startedAtTime ) ;
+        sh:resultMessage "De gekozen datum ligt in de toekomst. Gelieve deze na te kijken." ;
```

Comment:
The message is still pending, feedback requested in Jira

Reply:
We're going for "Let op: datum ligt in de toekomst. Controleer uw keuze." instead, so that should be updated here.